languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -39491,6 +39492,71 @@ though, and has since been slightly modified)-->
         </rule>
     </category>
     <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+        <rule id="TIRET_BAS" name="tiret bas erroné" default="temp_off"><!-- Refer to https://github.com/languagetool-org/languagetool/issues/1204#issuecomment-443842446 -->
+            <antipattern><!-- file name: https://github.com/languagetool-org/languagetool/issues/1535 -->
+                <token regexp="yes" spacebefore="no">[a-z-]+</token>
+                <token spacebefore="no">.</token>
+                <token spacebefore="no" regexp="yes">[a-z0-9]{2,7}</token>
+            </antipattern>
+            <antipattern>
+                <token regexp="yes">['‘"“]</token>
+                <token regexp="yes">&comp_mots;+_&comp_mots;+</token>
+                <token regexp="yes" spacebefore="no">['‘"“]</token>
+            </antipattern>
+            <antipattern>
+                <token regexp="yes" skip="2">['‘"“]</token>
+                <token regexp="yes">&comp_mots;+</token>
+                <token regexp="yes" spacebefore="no">&comp_mots;+</token>
+                <token regexp="yes" spacebefore="no">['‘"“]</token>
+            </antipattern>
+            <antipattern>
+                <token skip="3" regexp="yes">=|\*|\/|\+|\-|%</token>
+                <token regexp="yes">.*_.*</token>
+            </antipattern>
+            <antipattern>
+                <token skip="3" regexp="yes">/=|\*|\/|\+|\-|%)_</token>
```

Comment:
The `)` is unmatched. You need to escape it, or to add `(` previously.

Reply:
Thank you so much!!! I was going crazy yesterday because I couldn't find the error.

🙇